### PR TITLE
- add support for drawable overrides

### PR DESCRIPTION
**Pull request by james-postmates**
- update dependencies to latest versions (gradle and jetpack)
- add documentation for overriding the drawable characteristics
- increase version for public release to jitpack
- move all resource values into `public.xml` from their respective
  other files

### CODE REVIEW

File: centersliderview/build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (prerak-trivedi) [inline centersliderview/build.gradle]: we can remove this since there's no `jar` dependency.

File: centersliderview/build.gradle [hunk 2]
```diff
@@ -33,7 +33,7 @@ android {
...
 
-    implementation 'com.android.support:appcompat-v7:27.0.0'
+    implementation "androidx.annotation:annotation:1.0.2"
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
```
Comment (prerak-trivedi) [inline centersliderview/build.gradle]: should also update these to `androidx.text.xyz`

File: centersliderview/src/main/res/values/attrs.xml [hunk 3]
```diff
@@ -1,18 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- attributes for the CenterSliderView -->
```
Comment (prerak-trivedi) [inline centersliderview/src/main/res/values/attrs.xml:3]: can remove this empty resource file.

File: centersliderview/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -1,4 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="super_dark">@android:color/black</color>
```
Comment (prerak-trivedi) [inline centersliderview/src/main/res/values/colors.xml:3]: can remove this empty resource file.

**Review by prerak-trivedi — COMMENTED** [review]
Added a few comments

File: centersliderview/build.gradle (hunk 1 above)
Comment (james-postmates) [reply centersliderview/build.gradle]: sure

File: centersliderview/build.gradle (hunk 2 above)
Comment (james-postmates) [reply centersliderview/build.gradle]: I never actually wrote tests for this, but yeah, I can update.

File: centersliderview/src/main/res/values/attrs.xml (hunk 3 above)
Comment (james-postmates) [reply centersliderview/src/main/res/values/attrs.xml:3]: will do; I guess we can re-add if there's ever a reason to have a private `attrs.xml` file

File: centersliderview/src/main/res/values/colors.xml (hunk 4 above)
Comment (james-postmates) [reply centersliderview/src/main/res/values/colors.xml:3]: will do; I guess we can re-add if there's ever a reason to have a private `colors.xml` file

**Review by prerak-trivedi — APPROVED** [review]
lgtm

**Pull request merged**